projects/common/vpk180/system_bd.tcl

```diff
@@ -0,0 +1,14 @@
+###############################################################################
+## Copyright (C) 2023-2023 Analog Devices, Inc. All rights reserved.
```

Comment:
The license header should have only 2023, and not `2023-2023`. This applies to all files, except for the Makefile